hbase-client/src/main/java/org/apache/hadoop/hbase/client/RegionInfo.java

```diff
@@ -346,15 +346,23 @@ static TableName getTable(final byte [] regionName) {
     return parseRegionName(regionName)[1];
   }
 
-  @InterfaceAudience.Private
-  static boolean isEncodedRegionName(byte[] regionName) throws IOException {
+  /**
+   * Figure if the passed bytes represent an encoded region name or not.
+   * @param regionName A Region name either encoded or not.
+   * @return True if <code>regionName</code> represents an encoded name.
+   */
+  @InterfaceAudience.Private // For use by internals only.
```

Comment:
Can we move this method to another class?

Reply:
Where? This seems like most obvious place to look if you were looking for this sort of utility?

Reply:
RegionUtils? Maybe follow on to see if there's other related IA.Private methods in more public classes?